reflex/utils/processes.py

```diff
@@ -319,7 +339,7 @@ def run_process_with_fallback(args, *, show_status_message, fallback=None, **kwa
                 f"There was an error running command: {args}. Falling back to: {fallback_args}."
             )
             run_process_with_fallback(
-                fallback_args,
+                get_command_with_loglevel(fallback_args),
```

Comment:
do we need this again here? because this is a recursive call and we handle it on line 328

Reply:
Ah yes, that's absolutely right